content/learn/maintained.adoc

```diff
@@ -0,0 +1,65 @@
+---
+menu:
+  learn:
+    parent: Workflow
+title: Validated Patterns - Maintained Tier
+weight: 44
+aliases: /requirements/maintained/
+aliases: /requirements/validated/
+---
+
+:toc:
+
+= The Validated Patterns 'Maintained' Tier 
+
+[id="tldr"]
+== tl;dr
+
+The Maintained tier is intended to provide consumers with additional "sales" collateral and reassurance that the pattern was known to be functional on all currently supported LTS versions of OpenShift.  Inclusion in this tier may require additional work for the pattern's owner - which might be a partner or sufficiently motivated SME. 
+
+[id="nominating-a-community-pattern-to-become-validated"]
+== Nominating a Tested Pattern to become Maintained
+
+If there is a Tested pattern that you believe would be a good candidate for promotion to Maintained, you can make that case by emailing validatedpatterns@googlegroups.com
+
+Please be aware that each Maintained pattern represents an ongoing maintenance, support, and testing effort.  Finite team capacity means that it is simply not possible for the team to take on this responsibility for all Validated Patterns.  
+
+For this reason we have designed the tiers and our processes to facilitate this to occur outside of the team by any sufficiently motivated party, including other parts of Red Hat, partners, and even customers.
+
+In limited cases, the Validated Patterns team may consider taking on that work, however please get in contact at least 4 weeks prior to the end of a given quarter in order for the necessary work to be considered as part of the following quarter's planning process
+
+
+[id="requirements"]
+== Requirements
+
+Validated Patterns have deliverable and requirements *in addition* to those
+specified for the link:/requirements/tested/[Tested tier]
+
+[id="must"]
+=== Must
+
+. Maintained Patterns *MUST* continue to meet the following criteria to remain in Maintained Tested tier
+. Maintained Patterns *MUST* conform to the common technical link:/requirements/implementation/[implementation requirements]
+. Maintained Patterns *MUST* only make use of components that are either supported, or easily substitued for supportable equivalents (eg. HashiCorp vault which has community and enterprise variants)
+. Maintained Patterns *MUST NOT* rely on functionality in tech-preview, or hidden behind feature gates
+. Maintained Patterns *MUST* have their architectures reviewed by the PM, TPM, or TMM of each Red Hat product they consume to ensure consistency with the product teams` intentions and roadmaps
```

Comment:
Don't know, this feels like an especially heavy burden, not only on Pattern's authors, but also on the *PM/*TMM on the other side of the fence. Maybe a *should* is more appropriate here?

Reply:
Remember this is for *maintained* patterns. There should be fewer, and the burden should be higher.